src/rgw/rgw_coroutine.h

```diff
@@ -174,9 +174,8 @@ struct rgw_spawned_stacks {
   }
 
   void inherit(rgw_spawned_stacks *source) {
-    for (vector<RGWCoroutinesStack *>::iterator iter = source->entries.begin();
-         iter != source->entries.end(); ++iter) {
-      add_pending(*iter);
+    for (auto entry : source->entries) {
```

Comment:
Extra copy?

Reply:
no. entry is a pointer. but i s/auto/auto*/ for better readability.